site/src/atomics.template

```diff
@@ -0,0 +1,59 @@
+---
+title: Title
+description: Description
+template: standard
+---
+
+# Page title
+
+The first thing is the general description.
+
+Consider using a table to organize information.
+
+| cssproperty | value                           | screensize |
+| ----------- | ------------------------------- | ---------- |
+| `<name>`    | `<value>`, `<value>`, `<value>` | `<size>`   |
+
+## Usage
+
+Provide a focused example that displays this atomic's usage.
+
+```html
+<div>
+  <button class="button">Click me</button>
```

Comment:
Should the example here also be `border` to match the next two examples? 
- border
- border responsive
- border modifiers

Reply:
The previous example that was here before was updated per Will's request, so it does not seem like we **require** to have `border` in examples.